packages/story-editor/src/components/panels/design/videoOptions/useVideoTranscoding.js

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
Feels like this hook should exist [here](https://github.com/GoogleForCreators/web-stories-wp/tree/main/packages/story-editor/src/app/media/utils). It is used for multiple commpents and should not live with any component.

Reply:
The hook uses functionality from the story-editor package, so I didn't see an easy way to move to another package. We could extract that functionality, but that would make the hook less reusable (as the removed logic would have to be duplicated).

Reply:
Jonny pointed to `packages/story-editor/src/app/media/utils`, not another package. But not sure it's a great fit.

Reply:
Ah yes, I see now. Yeah, that could work. I have no strong opinions either way. It's only ever (according to current plans at least) going to be used in the two locations where it is now, so anything that works for those are ok with me.

Reply:
Moved in fc958e66c4.